commands/kafka/cluster/cluster.go

```diff
@@ -0,0 +1,34 @@
+package cluster
+
+import (
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	"github.com/spf13/cobra"
+)
+
+var (
+	allCols     = []string{"Id", "Name", "Version", "Size", "DatacenterId", "LanId", "BrokerAddresses", "State"}
+	defaultCols = []string{"Id", "Name", "Version", "Size", "DatacenterId", "LanId", "BrokerAddresses", "State"}
+)
+
+func Command() *core.Command {
+	cmd := &core.Command{
+		Command: &cobra.Command{
+			Use:              "cluster",
+			Short:            "The sub-commands of 'ionosctl kafka cluster' allow you to manage kafka clusters",
+			Aliases:          []string{"cl"},
+			TraverseChildren: true,
+		},
```

Comment:
```suggestion
	PersistentPreRun: func(cmd *cobra.Command, args []string) {
		location, _ := cmd.Flags().GetString(constants.FlagLocation)
		changeLocation(client.Must().KafkaClient, location)
	},
```

after moving Lcation as a  PersistentFlag on this command